.github/workflows/ci-unix-static-av2.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: CI Unix Static AV2
```

Comment:
Note: This is listed twice if the job name is not shown, because of the matrix below. Still better than before.

Reply:
Not sure which page you're referring to, but on the actions page at least, it's listed once, and when you click on it you can see the list of OSs, e.g. https://github.com/AOMediaCodec/libavif/actions/runs/6071494062
In any case there's not much we can do about that.